pdsfile/pds4file/__init__.py

```
@@ -13,28 +13,29 @@
-                                        r'_lien_resolution|)' +
-                                        r'((|_calibrated|_diagrams|_metadata|_previews)' +
-                                        r'(|_md5\.txt|\.tar\.gz))$')
+                                        r'_v[0-9]+\.[0-9]+)*$')
     BUNDLESET_PLUS_REGEX_I = re.compile(BUNDLESET_PLUS_REGEX.pattern, re.I)
+    BUNDLENAME_REGEX = re.compile(r'^([a-zA-z_].+)$')
 
-    BUNDLENAME_REGEX = re.compile(r'^(uranus_occ_u\d{0,4}._[a-z]*_(fos|\d{2,3}cm)|' +
-                                  r'cassini_[a-z]{3,4}_(cruise|saturn)|' +
+    BUNDLENAME_REGEX = re.compile(r'^(uranus_occ_u\d{0,4}._[a-z]*_(fos|\d{2,3}cm)(|_.*)|' +
+                                  r'cassini_[a-z]{3,4}_(cruise|saturn)(|_.*)|' +
                                   r'cassini_uvis_solarocc_beckerjarmak2023(|_.*))$')
```

Ruff (0.15.2)

[warning] Class field `BUNDLENAME_REGEX` is defined multiple times

Remove duplicate field definition for `BUNDLENAME_REGEX`

(PIE794)